src/depgraphs.nim

```diff
@@ -295,6 +306,7 @@ proc expand*(c: var AtlasContext; g: var DepGraph; nc: NimbleContext; m: Travers
         g.nodes[i].status = status
 
       if g.nodes[i].status == Ok:
+        g.nodes[i].nimbleFile = c.findNimbleFile(g.nodes[i].pkg, dest)
```

Comment:
Why?

Reply:
It's been a while but there were cases where the nimble file wasn't being found for newly added dependencies as it's traversing the graph. 

Also, it needs to properly set the full path for the nimble file. There's issues when the deps or projects folder are different from the standard locations.